Server/mods/deathmatch/logic/CStaticFunctionDefinitions.h

```diff
@@ -700,6 +700,7 @@ class CStaticFunctionDefinitions
     // Account set funcs
     static CAccount* AddAccount(const SString& strName, const SString& strPassword, bool bAllowCaseVariations, SString& strOutError);
     static bool      RemoveAccount(CAccount* pAccount);
+    static bool      SetAccountSerial(std::string strName, std::string strSerial);
```

Comment:
Please avoid using Hungarian notation in new code, `name` and `serial` would be neater

Reply:
adjusted 9ad1188